detect_secrets/core/usage.py

```diff
@@ -78,10 +127,36 @@ def add_console_use_arguments(self):
         return self
 
     def parse_args(self, argv):
-        output = self.parser.parse_args(argv)
-        PluginOptions.consolidate_args(output)
+        # We temporarily remove '--help' so that we can give the full
+        # amount of options after loading custom plugins.
+        argv_without_help = list(
+            filter(
+                lambda arg: (
+                    arg not in ('-h', '--help')
+                ),
+                argv,
+            ),
+        )
+        known_args, _ = self.parser.parse_known_args(
+            args=argv_without_help,
+        )
 
-        return output
+        # Audit does not use --custom-plugins
```

Comment:
Why not? I'm not sure if that's how I read this PR.

Reply:
I can amend to `# Audit does not use --custom-plugins, it pulls custom_plugins from the audited baseline`?

Reply:
++, this comment makes more sense